docs/sources/setup-grafana/upgrade-grafana.md

```diff
@@ -13,33 +13,33 @@ weight: 500
 
 # Upgrade Grafana
 
-We recommend that you upgrade Grafana often to stay up to date with the latest fixes and enhancements.
-In order to make this a reality, Grafana upgrades are backward compatible and the upgrade process is simple and quick.
+We recommend that you upgrade Grafana often to stay current with the latest fixes and enhancements.
+Because Grafana upgrades are backward compatible, the upgrade process is straightforward. Upgrading between many minor versions and one major version is generally safe, and dashboards and graphs will not change.
```

Comment:
"Upgrading between many minor versions" - not sure this reads well.  Are you saying upgrading between all minor versions of a major version is safe?  And it's usually safe to upgrade to the next major version?

Reply:
@tleopold , I'm actually not sure what this means. It's prose available in the original content. Should we strike it? My solution would be to word as follows: "Because Grafana upgrades are backward compatible, the upgrade process is straightforward, and dashboards and graphs will not change."

Reply:
Yes, this reads much better.